### PR TITLE
Set preload to false on most links

### PR DESCRIPTION
**Pull request by dan-mba**
## Changes proposed
The additional JavaScript loaded by preload causes performance issues on mobile.
The NextJS configuration of Webpack will not create a separate main chunk for links that are preloaded on desktop only.
This PR will set all links using our Link component to `preload={false}`.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7356"></a>

### CODE REVIEW

File: components/Link.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ export default function Link({ children, className, rel, ...restProps }) {
           ? className
           : "text-blue-600 underline decoration-dotted dark:text-blue-500 hover:underline hover:decoration-solid"
       }
+      prefetch={false}
```
Comment (eddiejaoude) [inline components/Link.js:12]: Do we want to have a condition so it only happens on mobile?

Comment (dan-mba) [reply components/Link.js:12]: I tried that, but webpack still bundles the code to preload the JavaScript files if the preload is set by a conditional. It is around 700k of JS that gets loaded by the main-XXX chunk of the bundle without preload set to false.

Comment (eddiejaoude) [reply components/Link.js:12]: ouch, ok thank you for clarifying 👍

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**